Dockerfile

```diff
@@ -26,4 +26,5 @@ RUN pip install --no-cache-dir /src/scraper \
 RUN mkdir -p /output
 WORKDIR /output
 
+ENV ZIMUI_DIST_DIR=/src/zimui OUTPUT_DIR=/output BUILD_DIR=/tmp
```

Comment:
Yes you can, I did not find much recommendations, since in any case `ENV` creates only a temporary layer (i.e. it is not persisted at the end of the build / pushed to the Docker repo).
If you prefer multiple lines, I won't argue.